synapse/replication/tcp/external_cache.py

```diff
@@ -94,7 +94,10 @@ async def set(self, cache_name: str, key: str, value: Any, expiry_ms: int) -> No
 
         logger.debug("Caching %s %s: %r", cache_name, key, encoded_value)
 
-        with opentracing.start_active_span("ExternalCache.set"):
+        with opentracing.start_active_span(
+            "ExternalCache.set",
+            tags={opentracing.SynapseTags: cache_name},
```

Comment:
I assume this should be?

```suggestion
            tags={opentracing.SynapseTags.CACHE_NAME: cache_name},
```

Reply:
:facepalm: